trigger/version.go

```diff
@@ -104,6 +104,11 @@ func (t *versionTriggers) makeData(sub *event.Subscription, pastTenseOverride st
 	projectName := t.version.Identifier
 	if api.ProjectIdentifier != nil {
 		projectName = utility.FromStringPtr(api.ProjectIdentifier)
+	} else {
+		identifier, err := model.GetIdentifierForProject(*api.Project)
```

Comment:
Same thing as above - seems like the [project ID used here](https://github.com/evergreen-ci/evergreen/blob/24b4506727504198dd5cbbd47fb020c3e7e0a89a/rest/model/version.go#L55) is [using the same logic to get the project identifier](https://github.com/evergreen-ci/evergreen/blob/24b4506727504198dd5cbbd47fb020c3e7e0a89a/rest/model/version.go#L81-L86) as the if block above this.

Reply:
removed